sdk/identity/identity/package.json

```diff
@@ -80,6 +80,9 @@
   },
   "devDependencies": {
     "@azure/abort-controller": "1.0.0-preview.2",
+    "@opencensus/core": "~0.0.17",
```

Comment:
Most dependencies should use `^`.  Is there a reason these new dependencies use `~` instead?